### PR TITLE
ADD hemantsw.json

### PR DESCRIPTION
**Pull request by hemant-sw**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1671"></a>

### CODE REVIEW

File: public/data/Hemantsw.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "YOUR NAME",
```
Comment (Panquesito7) [inline public/data/Hemantsw.json:2]: You need to add your name here.

```suggestion
  "name": "Hemant Sharma",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/Hemantsw.json (hunk 1 above)
Comment (hemant-sw) [reply public/data/Hemantsw.json]: Hey! @Panquesito7  I change the name please review  it.

**Comment by hemant-sw** [comment]
Name changed @Panquesito7

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! :tada:


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contributions :tada:

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

**Pull request merged**